src/components/inspect/PersonalPhraseInspector.js

```diff
@@ -6,9 +6,9 @@ import {editSession} from '../../actions'
 import './PersonalPhraseInspector.scss'
 
 const mapStateToProps = (state) => {
+  const { user, session } = state
```

Comment:
Same here, can be put into `state` instead, right?

Reply:
yeah :)